durn/actions.go

```diff
@@ -1,10 +1,36 @@
 package durn
 
 import (
-	_ "context"
-	_ "sync"
+	"context"
+	"fmt"
+	"regexp"
 
-	_ "durn2.0/auth"
-	_ "durn2.0/util"
-	_ "github.com/google/uuid"
+	db "durn2.0/database"
+	"durn2.0/models"
+	rl "durn2.0/requestLog"
+	"durn2.0/util"
 )
+
+// AddValidVoters checks that provided voters are entered as valid emails, and if that
+// is the case inserts them into the database
+func AddValidVoters(ctx context.Context, voters []models.Voter) error {
+	mailregex := "[a-zA-Z]+@kth\\.se"
```

Comment:
Are you sure that these are the only valid email addresses? I don't think I have seen one with numbers (or special symbols), but a few years back you were able to change your address. Maybe people could set them to something like 123@kth.se then?

Reply:
I have seen addresses with a number, for example `axelelm2` (doesn't exist)

Reply:
No, I am not sure that this is a good regex, so I was hoping for some second opinions. I googled a (little) bit trying to find some formal definition from kth, but did not get any. I could find a few email addresses containing numbers in zfinger when looking, so at least that should be included.

`.+@kth\.se` is probably good enough though, so maybe I'll change it to that?

Reply:
I guess that is good, Otherwise, [^@]*@kth\.se could be an alternative. Shouldn't really matter, but accepting an address like nv@tn@kth.se looks a bit ugly imo.

Reply:
yes, sounds good